src/app/components/Modals/UnbondModal.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  MobileDialog,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+import { useMediaQuery } from "usehooks-ts";
+
+import { screenBreakPoints } from "@/config/screen-breakpoints";
+
+interface UnbondModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onProceed: () => void;
+  awaitingWalletResponse: boolean;
+}
+
+export const UnbondModal = ({
+  isOpen,
+  onClose,
+  onProceed,
+  awaitingWalletResponse,
+}: UnbondModalProps) => {
+  const isMobileView = useMediaQuery(`(max-width: ${screenBreakPoints.md})`);
```

Comment:
@jeremy-babylonlabs lets create a custom hook for this `useBreakpoint(breakpoint: "md" | "lg" | ...)`